pulp.spec

```diff
@@ -407,6 +407,25 @@ A collection of components that are common between the pulp server and client.
 %doc README LICENSE COPYRIGHT
 
 
+# ---- Devel ------------------------------------------------------------------
+
+%package -n python-pulp-devel
+Summary: Pulp devel python packages
+Group: Development/Languages
+
+%description -n python-pulp-devel
+A collection of tools used for developing & testing Pulp plugins
+
+%files -n python-pulp-devel
+%defattr(-,root,root,-)
+%dir %{_usr}/lib/%{name}
```

Comment:
Does pulp.devel actually put things in /usr/lib/pulp? If not, we probably don't need this line.